tests/src/application/controllers/forget-password.spec.ts

```diff
@@ -1,9 +1,11 @@
 import { ForgetPasswordController } from '@/application/controllers/forget-password'
 import { type EmailValidator } from '@/application/validation/protocols'
-import { type TokenGenerator } from '@/data/protocols/recovery-password/token-generator'
+import { type TokenGenerator } from '@/data/protocols/cryptography'
 import { type ForgetPassword, type EmailService } from '@/domain/use-cases'
 import { makeEmailValidator, makeFakeGuardianWithIdData, makeFakeServerError, makeLoadGuardianByEmail, makeTokenGenerator } from '@/tests/utils'
 import { type LoadGuardianByEmailRepository } from '@/data/protocols'
```

Comment:
I believe it would be possible to import the token generator here in this line. What do you think?